docs/internal-documentation/maintenance/Instructions-for-translators.md

```diff
@@ -242,16 +243,18 @@ comments on these. The steps are the same for all three repositories.
   Github user name instead of "xxxx". There you can select the fork for this
   translation.
 
-* Select to create a new pull request. Using Zonemaster-Engine as an example,
-  the base repository should be *zonemaster/zonemaster-engine* and the base
-  branch should be *develop* (not *master*). The "head" should be your fork and
-  "compare" the same branch as you created above and pushed to your fork,
-  "translation-update".
+* Select to create a new pull request. Here we use Zonemaster-Engine as an
+  example.
+  * The base repository should be *zonemaster/zonemaster-engine*.
+  * The base branch should be *develop* (not *master*).
+  * The "head" should be your fork.
+  * The "compare" should be the same branch as you created above and pushed to
+    your fork, "translation-update".
```

Comment:
`"translation-update"` -> `e.g. "translation-update"`?

Sorry, I didn't see this before.